examples/product-sample-supabase-kt/app/src/main/java/com/example/manageproducts/di/SupabaseModule.kt

```diff
@@ -0,0 +1,55 @@
+package com.example.manageproducts.di
+
+import com.example.manageproducts.BuildConfig
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import io.github.jan.supabase.SupabaseClient
+import io.github.jan.supabase.createSupabaseClient
+import io.github.jan.supabase.gotrue.GoTrue
+import io.github.jan.supabase.gotrue.gotrue
+import io.github.jan.supabase.postgrest.Postgrest
+import io.github.jan.supabase.postgrest.postgrest
+import io.github.jan.supabase.storage.Storage
+import io.github.jan.supabase.storage.storage
+import javax.inject.Singleton
+
+
+@InstallIn(SingletonComponent::class)
+@Module
+object SupabaseModule {
+
+    @Provides
+    @Singleton
+    fun provideSupabaseClient(): SupabaseClient {
+        return createSupabaseClient(
+            supabaseUrl = BuildConfig.SUPABASE_URL,
+            supabaseKey = BuildConfig.API_KEY
+        ) {
+            install(Postgrest)
+            install(GoTrue)
```

Comment:
@phamhieu Suggested to use the PKCE authentication flow for the Google Login. This should do it:
```kotlin
install(GoTrue) {
    flowType = FlowType.PKCE
}
```
supabase-kt should handle the rest.

Reply:
Thank you for your suggestion. Just updated